test/jdk/java/lang/ProcessBuilder/InheritIO/InheritIo.java

```diff
@@ -23,12 +23,14 @@
 
 import static java.lang.ProcessBuilder.Redirect.*;
 
-class InheritIO {
+class InheritIo {
```

Comment:
It is just one class now so there is no need for package either. I ran a rebase on this branch since deleting it would delete this PR also, I presume. So I deleted the files related to other tests which now have their own PR ([1](https://github.com/openjdk/jdk/pull/1579), [2](https://github.com/openjdk/jdk/pull/1578), [3](https://github.com/openjdk/jdk/pull/1577)) and made a diff to `master` (`HEAD` being `692b273ec53f54a879a4bbaad6c2f5f1d5358a71`):
`
[JDK-8166026-refactor-shell-to-java] $ git diff --name-only  open-mainline/master
test/jdk/java/lang/ProcessBuilder/InheritIO/InheritIO.java
test/jdk/java/lang/ProcessBuilder/InheritIO/InheritIO.sh
test/jdk/java/lang/ProcessBuilder/InheritIOTest.java

`